lblod/frontend-form-builder
# SFB-112: Show message when there are unreferenced subjects

## ID
 [SFB-112](https://binnenland.atlassian.net/browse/SFB-112)

 ## Description

When removing a field there are some remains in the form of that field. These remains are removed by the `ember-submission-helpers` normally? So removing these is possible in the form-builder but this would be an overwrite of the data. The overwrite needs to be based on known information what we do not have we the remains are in the form. What we can do is that we show a toaster message when a subject is written in the form that is unrefereced. When a form is copy pasted in the form-builder and it will not remove unreferenced subjects but show you a mesage so you cna edit the code.

 ## Type of change


 ## How to test

1. Create a new form
2. Remove the field with the `builder`
3. the field is not in the preview anymore but the remains are still in the ttl code
4. You should get a toaster message on screen and a warn console log in the browsers console

 ## Links to other PR's

- /
